### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by reidbraswell1**
Ready to Merge On boarding project into the master branch. Ready for review.

### CODE REVIEW

File: OnboardingExperience/User.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        public string FirstName { get; set; }
+        public string Address { get; set; }
+        public string City { get; set; }
+        public string State { get; set; }
+        public int ZipCode { get; set; }
+        public bool IsAccountOwner { get; set; }
+        public int PINNumber { get; set; }
+
+        // Default Constructor
+        public User()
```
Comment (codytwinton) [inline OnboardingExperience/User.cs]: A default constructor can be inferred, so you can remove this.

File: OnboardingExperience/ValidatorHelper.cs [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                if (response.Equals("N") || response.Equals("n"))
+                    return false;
+                DisplayErrorText("Sorry I was unable to understand your response. Please use only Y(y) for yes or N(n) for no.");
+            }//while//
+        }//AskQuestionBool//
+
+        /*
+         * Build the output response.
+         */
+        public static StringBuilder BuildOutputResponse(User user)
```
Comment (codytwinton) [inline OnboardingExperience/ValidatorHelper.cs]: This is a fine implementation, but I'd suggest overriding the User `ToString()` method instead.

File: OnboardingExperience/ValidatorHelper.cs [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        {
+            DisplayUserOutput(errorText);
+        }//DisplayErrorText//
+
+        /*
+         * Helper method to get user input.
+         */
+        private static string GetUserInput()
+        {
+            return Console.ReadLine();
```
Comment (codytwinton) [inline OnboardingExperience/ValidatorHelper.cs]: This seems like unnecessary encapsulation. Generally a single line of a prebuilt method probably doesn't need an additional helper method.

File: OnboardingExperience/ValidatorHelper.cs [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+        }//getUserInput//
+
+        /*
+         * Helper method to display user output.
+         */
+        private static void DisplayUserOutput(string text)
+        {
+            Console.WriteLine(text);
```
Comment (codytwinton) [inline OnboardingExperience/ValidatorHelper.cs]: This seems like unnecessary encapsulation.

**Comment by reidbraswell1** [comment]
Changes have been made ready for review again.

File: OnboardingExperience/User.cs [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        public string State { get; set; }
+        public int ZipCode { get; set; }
+        public bool IsAccountOwner { get; set; }
+        public int PINNumber { get; set; }
+
+
+        /*
+         * Build the output response.
+         */
+        public override string ToString()
```
Comment (codytwinton) [inline OnboardingExperience/User.cs]: Well done.

**Pull request merged**